README.md

```diff
@@ -229,14 +236,10 @@ Running the bos-workspace dev server will start a local gateway with a standard
 bw dev --no-gateway
 ```
 
-However, there is an option to override this default gateway with a custom `/dist`. This is helpful when building widgets that utilize [custom VM elements](https://github.com/NEARBuilders/near-bos-webcomponent?tab=readme-ov-file#configuring-vm-custom-elements). To use this feature, use the `-g` flag with a path to the local custom distribution or link to package published on [nearfs](https://github.com/vgrichina/nearfs) or via cdn:
-
-```cmd
-bw dev -g path/to/dist
-```
+However, there is an option to override this default gateway with a custom `/dist`. This is helpful when building widgets that utilize [custom VM elements](https://github.com/NEARBuilders/near-bos-webcomponent?tab=readme-ov-file#configuring-vm-custom-elements). To use this feature, use the `-g` flag and specify the gateway url and the tag name in the `bos.config.json` file.
```

Comment:
"To use this feature, specify the gateway bundle url and the tag name in the `bos.config.json` file."